src/amalgam/gen/rvv.c

```diff
@@ -1517,6 +1517,28 @@ void xnn_f32_vcmul_ukernel__rvv_u2v(
   }
 }
 
+void xnn_f32_vlrelu_ukernel__rvv_u4v(
+    size_t batch,
+    const float* input,
+    float* output,
+    const union xnn_f32_lrelu_params params[restrict XNN_MIN_ELEMENTS(1)])
+{
+  assert(batch != 0);
+  assert(batch % sizeof(float) == 0);
+  assert(input != NULL);
+  assert(output != NULL);
+
+  batch >>= XNN_LOG2_SIZEOF_FLOAT;
+
+  do {
+    size_t n = __riscv_vsetvl_e32m4(batch); batch -= n;
+    vfloat32m4_t in_f32v = __riscv_vle32_v_f32m4(input, n); input += n;
+    vbool8_t mask_f32v = __riscv_vmflt_vf_f32m4_b8(in_f32v, 0.0f, n);
+    vfloat32m4_t out_f32v = __riscv_vfmul_vf_f32m4_mu(mask_f32v, in_f32v, in_f32v, params->scalar.slope, n);
```

Comment:
Suggest to define `const float vslope = params->scalar.slope;` outside the do_while.
Also, you can replace `params->scalar.slope` with `vslope` here.

Reply:
Done